Content.Server.Database/Migrations/Postgres/PostgresServerDbContextModelSnapshot.cs

```diff
@@ -283,8 +283,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                         .HasColumnType("timestamp with time zone")
                         .HasColumnName("expiration_time");
 
-                    b.Property<DateTime?>("LastEditedAt")
-                        .IsRequired()
+                    b.Property<DateTime>("LastEditedAt")
```

Comment:
This is an auto-generated file, and EF Core 9 seems to generate it a little differently than 8. Not my doing, so let me know I need to do something about it.